prettyprinter-configurable/src/Text/Fixity/Internal.hs

```diff
@@ -205,74 +206,85 @@ Implementing all of that is left as future work.
-    { _renderContextDirection :: !Direction
-    , _renderContextFixity    :: !(FixityOver prec)
-    } deriving stock (Show, Eq)
+  { _renderContextDirection :: !Direction
+  , _renderContextFixity    :: !(FixityOver prec)
+  }
+  deriving stock (Show, Eq)
 
 -- Instead of receiving a @a -> a@ this function could simply return a 'Bool'.
--- | Enclose an @a@ (using the provided function) if required or leave it as is.
--- The need for enclosing is determined from an outer 'RenderContext' and the inner fixity.
+
+{-| Enclose an @a@ (using the provided function) if required or leave it as is.
+The need for enclosing is determined from an outer 'RenderContext' and the inner fixity.
+-}
 encloseIn
-    :: Ord prec
-    => (a -> a)                -- ^ Enclose a value of type @a@ in parens.
-    -> RenderContextOver prec  -- ^ An outer context.
-    -> FixityOver prec         -- ^ An inner fixity.
-    -> a
-    -> a
+  :: (Ord prec)
+  => (a -> a)
+  -- ^ Enclose a value of type @a@ in parens.
+  -> RenderContextOver prec
+  -- ^ An outer context.
+  -> FixityOver prec
+  -- ^ An inner fixity.
+  -> a
+  -> a
 encloseIn parens (RenderContext dir (Fixity assocOut precOut)) (Fixity assocInn precInn) =
-    case precOut `compare` precInn of
-        LT -> id                       -- If the outer precedence is lower than the inner, then
-                                       -- do not add parens. E.g. in @Add x (Mul y z)@ the precedence
-                                       -- of @Add@ is lower than the one of @Mul@, hence there is
-                                       -- no need for parens in @x + y * z@.
-        GT -> parens                   -- If the outer precedence is greater than the inner, then
-                                       -- do add parens. E.g. in @Mul x (Add y z)@ the precedence
-                                       -- of @Mul@ is greater than the one of @Add@, hence
-                                       -- parens are needed in @x * (y + z)@.
-        EQ -> case (assocOut, dir) of  -- If precedences are equal, then judge from associativity.
-            _ | assocOut /= assocInn       -> parens  -- Associativities differ => parens are needed.
-            (LeftAssociative, ToTheLeft)   -> id      -- No need for parens in @Add (Add x y) z@
-                                                      -- which is rendered as @x + y + z@.
-            (RightAssociative, ToTheRight) -> id      -- No need for parens in @Concat xs (Concat ys zs)@
-                                                      -- which is rendered as @xs ++ ys ++ zs@.
-            _                              -> parens  -- Every other case requires parens.
+  case precOut `compare` precInn of
+    LT -> id -- If the outer precedence is lower than the inner, then
+    -- do not add parens. E.g. in @Add x (Mul y z)@ the precedence
+    -- of @Add@ is lower than the one of @Mul@, hence there is
+    -- no need for parens in @x + y * z@.
```

Comment:
It was nice and readable and now it's just utterly disgusting. Could you please return the old version (modulo indentation) and turn off `fourmolu` in this file? Oh, and please remove the stupid newline from the comment above.